### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by JorikSchellekens**

### CODE REVIEW

File: src/hardhat/tasks.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  for (const sourcepath of sourcePathsWarp) {
+    const contractNames = await getContractNames(sourcepath);
+    contractNames.map((contractName) => {
+      const contractObj = new ContractInfo(contractName, sourcepath);
+      saveContract(contractObj);
+    });
+  }
+});
+
+// subtask(TASK_COMPILE_WARP_RUN_BINARY)
```
Comment (swapnilraj) [inline src/hardhat/tasks.ts:66]: delete

File: src/ethers/ContractFactory.ts [hunk 2]
```diff
@@ -64,52 +55,49 @@ export class ContractFactory {
...
 
-    const inputs = encode(
-      this.interface.deploy.inputs,
-      args,
-    )
+    const inputs = encode(this.interface.deploy.inputs, args);
 
     const starknetContract = await this.starknetContractFactory.deploy(inputs);
-    console.log("deploying", this.pathToCairoFile);
+    console.log('deploying', this.pathToCairoFile);
```
Comment (swapnilraj) [inline src/ethers/ContractFactory.ts:83]: delete

File: src/ethers/ContractFactory.ts [hunk 3]
```diff
@@ -64,52 +55,49 @@ export class ContractFactory {
...
     console.log(starknetContract.deployTransactionHash);
```
Comment (swapnilraj) [inline src/ethers/ContractFactory.ts:84]: delete

File: src/ethers/ContractFactory.ts [hunk 4]
```diff
@@ -64,52 +55,49 @@ export class ContractFactory {
...
     await starknetContract.deployed();
     const contract = new WarpContract(
       starknetContract,
-      this.starknetContractFactory,
       this.ethersContractFactory,
-      this.pathToCairoFile
+      this.pathToCairoFile,
     );
-    console.log("deployed");
+    console.log('deployed');
```
Comment (swapnilraj) [inline src/ethers/ContractFactory.ts:91]: delte console log

File: src/transpiler.ts [hunk 5]
```diff
@@ -1,43 +1,27 @@
...
-
   public transpile(inputPath: string) {
     console.log(`Compiling: ${inputPath}`);
-    console.log(this._pathToWarp)
-    execSync(`${this._pathToWarp} transpile ${inputPath} --compile-cairo`, {stdio: 'inherit'});
+    console.log(this._pathToWarp);
+    execSync(`${this._pathToWarp} transpile ${inputPath} --compile-cairo`, { stdio: 'inherit' });
   }
 
   public async deploy(contractPath: string, parameters: string, testnet: boolean, wallet: string) {
```
Comment (swapnilraj) [inline src/transpiler.ts:11]: delete redundant deploy function

File: src/hardhat/tasks.ts (hunk 1 above)
Comment (JorikSchellekens) [reply src/hardhat/tasks.ts:66]: Keeping this around until we actually implement the warp compilation

**Pull request merged**